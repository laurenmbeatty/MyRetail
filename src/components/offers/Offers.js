import React from "react";
import * as Styled from "./styles";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
class Offers extends React.Component {
  render() {
    const { offers } = this.props;
    return (
      <div>
        <Styled.Rule />
        <Styled.Offers>
          {offers.map((offer, index) => (
            <li key={index}>
              <FontAwesomeIcon icon="tag" />
              <span>{offer.Description[0].shortDescription}</span>
            </li>
          ))}
        </Styled.Offers>
        <Styled.Rule />
      </div>
    );
  }
}
export default Offers;
