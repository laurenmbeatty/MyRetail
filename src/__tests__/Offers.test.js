import React from "react";
import { create } from "react-test-renderer";
import Offers from "../components/offers/Offers";

test("snapshot", () => {
  const c = create(<Offers />);
  expect(c.toJSON()).toMatchSnapshot();
});
