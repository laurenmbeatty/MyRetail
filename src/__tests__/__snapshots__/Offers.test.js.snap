// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <hr
    className="sc-bdVaJa htMPpU"
  />
  <hr
    className="sc-bdVaJa htMPpU"
  />
</div>
`;
